beekeeper-integration-tests/src/test/java/com/expediagroup/beekeeper/integration/BeekeeperCleanupIntegrationTest.java

```diff
@@ -268,10 +270,13 @@ void metrics() throws SQLException {
   }
 
   private void assertMetrics() {
-    MeterRegistry meterRegistry = BeekeeperCleanup.meterRegistry();
-    List<Meter> meters = meterRegistry.getMeters();
-    assertThat(meters).extracting("id", Meter.Id.class).extracting("name")
-        .contains("cleanup-job", "s3-paths-deleted", "s3-" + BytesDeletedReporter.METRIC_NAME);
+    Set<MeterRegistry> meterRegistry = ((CompositeMeterRegistry) BeekeeperCleanup.meterRegistry()).getRegistries();
+    assertThat(meterRegistry).containsExactlyInAnyOrder();
```

Comment:
Will this line work along with the following line? This one checks that there are no elements.

Reply:
Fixed